src/fides/api/ops/service/privacy_request/email_batch_service.py

```diff
@@ -130,6 +130,8 @@ def initiate_scheduled_batch_email_send() -> None:
     if CONFIG.test_mode:
         return
 
+    assert scheduler.running, "Scheduler is not running! Cannot add Batch Email job."
```

Comment:
@pattisdr added a simple check here, this feels like the right amount of protection to me